src/main/kotlin/io/pivotal/intellij/jasmine/JasmineRunProfileState.kt

```diff
@@ -72,20 +67,33 @@ class JasmineRunProfileState(private var project: Project,
         return SMTestRunnerConnectionUtil.createConsole("Jasmine", props)
     }
 
-    private fun jasminePath(runConfig: JasmineRunConfiguration): String{
+    private fun jasminePath(runConfig: JasmineRunConfiguration): String {
         val jasminePath = Paths.get(runConfig.selectedJasminePackage().systemDependentPath)
-                .resolve(runConfig.jasmineRunSettings.jasmineExecutable)
+            .resolve(runConfig.jasmineRunSettings.jasmineExecutable)
         return jasminePath.toAbsolutePath().toString()
     }
 
-    private fun findReporterPath(): String{
-        val jarPath = File(PathUtil.getJarPathForClass(this.javaClass))
-        val pluginRoot = jarPath.parentFile.parentFile
-        val reporterPath = FileUtil.toSystemDependentName("lib/intellij_reporter.js")
-        return File(pluginRoot, reporterPath).absolutePath
+    private fun findReporterPath(): String {
+        val inputStream = javaClass.classLoader.getResourceAsStream("intellij_reporter.js")
```

Comment:
These changes are because I couldn't find an idiomatic example of what prepareSandbox did in the previous version. 
The resources are on the classpath, so this creates a temp file. I have no idea if this is the right way to do it (seemed complex), but it worked for me™